<?xml version='1.0' encoding='utf-8'?>
<!--This does not replace URDF, and is not an extension of URDF.
    This is a format for representing semantic information about the robot structure.
    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
-->
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="panda">
  <xacro:include filename="$(find simple_panda_moveit_config)/config/panda_arm.xacro" />
  <xacro:panda_arm />

  <!--END EFFECTOR: Purpose: Represent information about an end effector.-->

  <!--
  <end_effector name="camera" parent_link="panda_link8" group="hand" parent_group="panda_arm" />
-->
  <!--DISABLE COLLISIONS: By default it is assumed that any link of the robot could potentially come into collision with any other link in the robot. This tag disables collision checking between a specified pair of links. -->

<disable_collisions link1="panda_d435i_bottom_screw_frame" link2="panda_link8" reason="Adjacent" />
<disable_collisions link1="panda_d435i_bottom_screw_frame" link2="panda_link7" reason="Default" />
<disable_collisions link1="panda_d435i_link"               link2="panda_link7" reason="Default" />
</robot>
